jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-api/src/main/java/org/jbpm/console/ng/wi/dd/model/DeploymentDescriptorModel.java

```diff
@@ -245,35 +245,20 @@ public boolean equals(Object o) {
     @Override
     public int hashCode() {
         int result = persistenceUnitName != null ? persistenceUnitName.hashCode() : 0;
-        result = ~~result;
```

Comment:
Nooooooo!

We started adding those operators as we found HashCode would behave really
strangely sometimes in GWT compiled code. On investigation GWT uses the ~~
operator extensively in its JRE emulated libraries' HashCode... so we
followed and our HashCode implementations behaved as expected.

Any class destined for the client side should, in my opinion use the ~~
operator in its HashCode implementation... or we'll return to having
"really hard to fix" bugs come back!